packages/headless/src/database/mongodb/user/user.model.ts

```diff
@@ -0,0 +1,33 @@
+import { randomUUID } from 'crypto';
+import { model, Schema } from 'mongoose';
+import { UserEntity } from 'src/entity/user';
+
+/**
+ * User Schema
+ */
+const UserSchema = new Schema<UserEntity>({
+  id: {
+    type: String,
+    unique: true,
+    default: () => randomUUID(),
+  },
+  firstName: String,
+  lastName: String,
+  displayName: String,
+  username: String,
+  email: {
```

Comment:
If the email has not any other property then you may use one line of this like email: String.